index.html

```diff
@@ -0,0 +1,279 @@
+              <div class="portfolio__img__content">
+              <h3 class="portfolio__tonic">Multi-Post Stories</h3>
+              <div class="feature__container">
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <div class="skills">
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </div>
+              </div>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <div class="Portfolio__project__content">See Project </div>
+                </div>
+              </div>
+            </div>
+            </div>
+            </div>
+          </section>
+          <section class="Portfolio__About__MySelf"> 
+            <div class="portfolio__card__myself">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <h3 class="About__MySelf__Connect">LET'S CONNECT</h3>
+              </div>
+              <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+           </div>
+           <div class="resume__tag">
+            <button class="resume__button"><p>Get my resume</p></button>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you style your project requirements to match the Figma Design.


| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="91" alt="image" src="https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/52934128/134d8f50-32e8-467d-9273-9a3380c7b365">  | <img width="123" alt="image" src="https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/52934128/e5d8280c-1535-401c-aecc-49edc303b303">  |